src/doc/book/structs.md

```diff
@@ -18,7 +19,7 @@ struct Point {
 }
 
 fn main() {
-    let origin = Point { x: 0, y: 0 }; // origin: Point
+    let origin = Point { x: 0, y: 0 }; // create an instance
```

Comment:
This is a convention we've been using throughout the starting bits of the book, to annotate the types of things.